templates/README.md

```diff
@@ -0,0 +1,38 @@
+# How to setup AWS Credentials rotation
+
+These files have to be deployed in the platform namespace.
+
+- prerequisite
+
+  ```text
+  - you have access to kubernetes cluster hosting 'platform' namespace.
+  - docker image 'nuxeo/aws-iam-credential-rotate' is available.
+  - create or retrieve credentials from aws:
+    - Access key ID
+    - Secret access key
+  ```
+
+## Create Secret
+
+[secret](secret.yaml) file will be created under the 'platform' namespace:
+
+- aws-iam-user-credentials secret
```

Comment:
I won't mention the secret name as it could change. Furthermore I'm not sure about the benefit to have it here as I can click on secret link and see the name.

Reply:
will be take to account